src/main/java/gui/RootCenterArea.java

```diff
@@ -1,62 +1,155 @@
+        mainTimelineScrollpane.setHbarPolicy(ScrollPane.ScrollBarPolicy.NEVER);
+        getChildren().add(mainTimelineScrollpane);
+    }
 
-        parentPane.setLeftAnchor(grid, 0.0);
-        parentPane.setRightAnchor(grid, 0.0);
-       // parentPane.setBottomAnchor(grid, 0.0);
-        parentPane.setTopAnchor(grid, 0.0);
+    /**
+     * Initializes the counter timeline to the left.
+     */
+    private void initCounterPane() {
+        counterScrollpane = new ScrollPane();
+        counterAnchorPane = new AnchorPane();
+        counterGridPane = new CounterGridPane(numberOfCounts, 100, 1000);
+        counterAnchorPane.setLeftAnchor(counterGridPane, 0.0);
+        counterAnchorPane.setRightAnchor(counterGridPane, 0.0);
+        counterAnchorPane.setTopAnchor(counterGridPane, 0.0);
+        counterAnchorPane.getChildren().add(counterGridPane);
+        counterScrollpane.setContent(counterAnchorPane);
+        counterScrollpane.setFitToWidth(true);
+        counterScrollpane.setHbarPolicy(ScrollPane.ScrollBarPolicy.NEVER);
+        counterAnchorPane.setMinHeight(1000);
+        counterScrollpane.setStyle("-fx-background-color: yellow"); // debugcolor for now
+        counterScrollpane.vvalueProperty().bindBidirectional(
+                mainTimelineScrollpane.vvalueProperty());
+        counterAndDirectorPane.getChildren().add(counterScrollpane);
+    }
 
-        parentPane.getChildren().add(grid);
-        parentPane.setMinHeight(1000);
-        parentPane.setMinWidth(1000);
+    /**
+     * Initializes the director timeline to the left.
+     */
+    private void initDirectorPane() {
+        directorScrollpane = new ScrollPane();
+        directorAnchorPane = new AnchorPane();
+        directorGridPane = new DirectorGridPane(numberOfCounts, 100, 1000);
+        directorAnchorPane.setLeftAnchor(directorGridPane, 0.0);
+        directorAnchorPane.setRightAnchor(directorGridPane, 0.0);
+        directorAnchorPane.setTopAnchor(directorGridPane, 0.0);
+        directorAnchorPane.getChildren().add(directorGridPane);
+        directorScrollpane.setContent(directorAnchorPane);
+        directorScrollpane.setFitToWidth(true);
+        directorScrollpane.setHbarPolicy(ScrollPane.ScrollBarPolicy.NEVER);
+        directorAnchorPane.setMinHeight(1000);
+        directorScrollpane.setStyle("-fx-background-color: orange"); // debugcolor for now
+        directorScrollpane.vvalueProperty().bindBidirectional(
+                mainTimelineScrollpane.vvalueProperty());
+        counterAndDirectorPane.getChildren().add(directorScrollpane);
+    }
 
-        setFitToWidth(true);
-        //setFitToHeight(true);
-        setContent(parentPane);
+    /**
+     * Initializes the scrollbar for the main timeline.
+     */
+    private void initScrollbar() {
+        ScrollBar scrollbar = new ScrollBar();
+        scrollbar.setMin(0);
+        scrollbar.setMax(1);
+        scrollbar.maxWidthProperty().bind(widthProperty().subtract(200));
```

Comment:
This depends on the min width for the directortimeline i suppose, extra reason to use constants